### PR TITLE
Feat/allow multiple extension requests

### PR DESCRIPTION
**Pull request by MayurLund**
**Changes Made:**
With this pull request, I have implemented dynamic button enabling for the "Create" button in accordance with the extension request status. The behavior is as follows:

**If no extension request exists previously:**
The "Create" button is now enabled, allowing users to initiate a new extension request.

**If there is a previous extension request:**
The "Create" button is enabled only if the status of the previous extension request is either "Approved" or "Denied."

Video for reference first that current behaviour is shown and after that with dev flag new behaviour is shown: 

[screen-capture (10).webm](https://github.com/Real-Dev-Squad/website-my/assets/37043221/5459605d-b358-40da-8e0b-ceb5bcf68397)

### Test Results & TestCoverage :
image

image
image

### CODE REVIEW

File: app/components/task/multiple-extension-form.js [hunk 1]
```diff
@@ -0,0 +1,172 @@
...
+      this.toast.error(error.message, '', {
+        ...toastNotificationTimeoutOptions,
+        timeOut: '3000',
+      });
+      setTimeout(this.args.closeModel, 2000);
+    }
+  }
+
+  @action
+  changeExtensionRequestETA(e) {
```
Comment (iamitprakash) [inline app/components/task/multiple-extension-form.js:158]: e.target.value property is the value of the input element that the event occurred on. In this case, it is the newEndsOn input element. However, the code is setting the e.target.value property to an empty string even if the extensionTime is valid. This means that the user will lose their input if the new ETA is valid.

**Review by iamitprakash — CHANGES_REQUESTED** [review]
Please make necessary arrangements

**Review by iamitprakash — CHANGES_REQUESTED** [review]
please ass working proof

**Review by iamitprakash — CHANGES_REQUESTED** [review]
please add working proof

File: app/components/task/multiple-extension-form.js (hunk 1 above)
Comment (MayurLund) [reply app/components/task/multiple-extension-form.js:160]: Done. Removed that empty string now user is not losing data.

**Comment by MayurLund** [comment]
> Please make necessary arrangements
Made the necessary changes suggested by you. Please review.

**Comment by MayurLund** [comment]
Hi @iamitprakash  @prakashchoudhary07  , I have made the requested changes in the pull request. Kindly review at your earliest convenience.

**Review by iamitprakash — APPROVED** [review]

**Pull request merged**